includes/sql.js

```diff
@@ -62,4 +62,11 @@ function percentChangeColumn(metric, granularity){
     return result;
 }
 
-module.exports = {multiColumnEqualsClause, parseIsoWeekYear, isoYearWeekColumn, percentChangeColumn, dashboardWebBrowsers, webBrowserCaseStatement};
\ No newline at end of file
+function aggregatedVBBColumns(eventDateField, eventDateAlias, columns) {
+    let result = `${eventDateField} as ${eventDateAlias}`;
+    for(const event_type in columns) {
+        result = result + `,\n  COUNTIF(event_name = "${event_type}") OVER(PARTITION BY ${eventDateField}) as ${columns[event_type]}`;
```

Comment:
Another reason to have incremental table - PARTITION BY are expensive.

Reply:
Expensive in terms of what? We're partitioning over 1 year, 365 days.

Reply:
It has to scan the whole base table to calculate the totals; partitioning adds complexity to the calculations. But if it's very infrequent then it probably doesn't matter.